src/options-resolver.ts

```diff
@@ -36,5 +36,12 @@ export const OptionsChoiceResolver = (options: any,
                                       choices: string[],
                                       readlineDependency?: any) => {
     const readline = readlineDependency || readlineSync;
-    return options[key] !== undefined ? options[key] : (readline.keyInSelect(choices, promptText));
+    if (options[key] !== undefined) {
+        return options[key];
+    }
+    const choiceIndex = readline.keyInSelect(choices, promptText);
+    if (-1 === choiceIndex) {
```

Comment:
Not aware of which issue is solving this change, could you explain the rationale behind it?

Reply:
issue #169